.gitignore

```diff
@@ -1,5 +1,5 @@
 # MegaLinter
```

Comment:
Commit message: it would be more accurate to say something like: "Replace MegaLinter v5 with v6 hooks."